addons/im_livechat/static/src/js/ajax_external.js

```diff
@@ -0,0 +1,7 @@
+odoo.define('web.ajax_external', function (require) {
+"use strict";
+
+var ajax = require('web.ajax');
+ajax.loadJS = function () {};
```

Comment:
This is missing a small explanation, something like:

```
/** 
  * This file should be used in the context of an external widget loading (e.g: live chat in a non-Odoo website)
  * It overrides the 'loadJS' method that is supposed to load additional scripts, based on a relative URL (e.g: '/web/webclient/locale/en_US')
  * As we're not in an Odoo website context, the calls will not work, and we avoid a 404 request.
  */
```

Also, I would add a `console.warn` in the method instead:

```
ajax.loadJS = function (url) {
    console.warn('Tried to load the following script on an external website: ' + url);
};
```

As we could miss some features by ignoring this call.